🔨 comments are instantaneous

zhiyongggggg opened the pull request.

zhiyongggggg committed:
🔨 comments are instantaneous

joshualwf commented:
some bugs:
- when i post a comment, i cannot like my own comment - need to refresh then it will work
- Lets say theres 3 comments, comment A, comment B and comment C, if i were to give comment B a like, then i post a comment, it wont reflect that comment B is the one that is liked, theres a ui bug

joshualwf commented:
also btw i changed the order of the getComments to be latest at the top

joshualwf committed:
changed getComments api to make latest comments at the top instead

zhiyongggggg committed:
🔨  fixed "when i post a comment, i cannot like my own comment - need to refresh then it will work"

zhiyongggggg committed:
🔨 one API call that will fetch liked status of comments

zhiyongggggg committed:
🔨 fixed issue

issue: "comment A, comment B and comment C, if i were to give comment B a like, then i post a comment, it wont reflect that comment B is the one that is liked, theres a ui bug"

zhiyongggggg committed:
🔨  Admin page for bulk upload

joshualwf commented:
bug: unable to view comments if im not logged in

joshualwf commented:
also rmb to make sure the error handling is such that when im not logged in and i try to comment/ do anything the toast will say "pls login etc" instead of "internal server error"

idk whats the current behaviour uh i j write here in case haha

zhiyongggggg committed:
🔨  fixed cannot view comments if not logged in

zhiyongggggg committed:
🔨  Delete comments

joshualwf committed:
update comment ui

joshualwf commented:
ive updated the delete ui

bugs:
- for nested replies: unable to like & delete
- npm run build errors

joshualwf committed:
update toast wordings

zhiyongggggg committed:
🔨  fixed issues

fixed:
1."for nested replies: unable to like & delete"
2. "npm run build errors"
